### PR TITLE
added common filters. Closes #110

### LINKED ISSUE #110: Default Filters
- Create configuration for default filters in innkeeper.
- Allow the user to append them, prepend them to the existing filters
- Allow the users to skip the default filters
- By default, prepend the filters

### PR DESCRIPTION
**Pull request by NorthFury**

### CODE REVIEW

File: src/it/scala/org/zalando/spearheads/innkeeper/routes/RoutesRepoHelper.scala [hunk 1]
```diff
@@ -89,4 +87,17 @@ object RoutesRepoHelper extends DaoHelper {
         |      }]
         |  }]
         |}""".stripMargin
+
+  private def insertTestPath(ownedByTeam: String, createdBy: String, createdAt: LocalDateTime): Long = {
```
Comment (danpersa) [inline src/it/scala/org/zalando/spearheads/innkeeper/routes/RoutesRepoHelper.scala]: you could split the parameters on more lines :) maybe this should be the default...

File: src/test/resources/common-filters.conf [hunk 2]
```diff
@@ -0,0 +1,10 @@
+filters.common {
+  prepend = [
+    "prepend-first"
```
Comment (danpersa) [inline src/test/resources/common-filters.conf]: in eskip, filters have the following format for eg: `Traffic("hello", 1.2)`

Comment (danpersa) [inline src/test/resources/common-filters.conf]: maybe the test should reflect this

File: src/test/scala/org/zalando/spearheads/innkeeper/services/team/ZalandoTeamServiceSpec.scala [hunk 3]
```diff
@@ -32,9 +32,8 @@ class ZalandoTeamServiceSpec extends FunSpec with MockFactory with Matchers with
       val teamService = new ZalandoTeamService(mockConfig, mockHttpClient)
 
       val now = LocalDateTime.now()
-      val route = RouteOut(1, 1L, RouteName("name"), NewRoute(
-        predicates = Some(Seq(
-          Predicate("somePredicate", Seq(StringArg("Hello"), NumericArg("123.0")))))), createdAt = now, activateAt = now, TeamName("pathfinder"), UserName("gigel"))
+      val newRoute = NewRoute(predicates = Some(Seq(Predicate("somePredicate", Seq(StringArg("Hello"), NumericArg("123.0"))))))
```
Comment (danpersa) [inline src/test/scala/org/zalando/spearheads/innkeeper/services/team/ZalandoTeamServiceSpec.scala]: These two can also be put on many lines

File: src/it/scala/org/zalando/spearheads/innkeeper/routes/RoutesRepoHelper.scala (hunk 1 above)
Comment (NorthFury) [inline src/it/scala/org/zalando/spearheads/innkeeper/routes/RoutesRepoHelper.scala]: for 3 or less parameters I think the readability is better if we keep them on the same line

**Comment by danpersa** [comment]
lgtm 👍

**Pull request merged**